### DISCUSSION TITLE
[info] in Container does not support markdown syntax

### DISCUSSION THREAD

**Discussion by tadmi** [vuepress-theme-hope/vuepress-theme-hope#2964] [Q&A] [answered]
```
::: info info _header_
container _text_
:::
```

![image](https://user-images.githubusercontent.com/63556979/228594841-e5f1ade5-428e-400e-bebb-45f392065686.png)

**Comment by Mister-Hope** [comment, marked as answer]
This is how markdown containers work, the text is just a text used as title
